src/main/java/org/apache/commons/net/util/SubnetUtils.java

```diff
@@ -223,136 +324,39 @@ public int getAddressCount() {
         }
 
         /**
-         * Get the count of available addresses.
-         * Will be zero for CIDR/31 and CIDR/32 if the inclusive flag is false.
-         * @return the count of addresses, may be zero.
-         * @since 3.4
+         * Returns the count of available addresses.
+         * Will be zero for CIDR/31 and CIDR/32 if the address is IPv4 address and
+         * the inclusive flag is <code>false</code>.
+         *
+         * @return the count of addresses, may be zero
          */
-        public long getAddressCountLong() {
-            long b = broadcastLong();
-            long n = networkLong();
-            long count = b - n + (isInclusiveHostCount() ? 1 : -1);
-            return count < 0 ? 0 : count;
-        }
-
-        public int asInteger(String address) {
-            return toInteger(address);
-        }
+        public long getAddressCountLong() { return 0; }
 
-        public String getCidrSignature() {
-            return format(toArray(address)) + "/" + pop(netmask);
-        }
+        /**
+         * Returns the count of available addresses.
+         * Will be zero for CIDR/31 and CIDR/32 if the address is IPv4 address and
+         * the inclusive flag is <code>false</code>.
+         *
+         * @return the count of addresses in a string, may be zero
+         */
+        public String getAddressCountString() { return null; }
 
-        public String[] getAllAddresses() {
-            int ct = getAddressCount();
-            String[] addresses = new String[ct];
-            if (ct == 0) {
-                return addresses;
-            }
-            for (int add = low(), j=0; add <= high(); ++add, ++j) {
-                addresses[j] = format(toArray(add));
-            }
-            return addresses;
-        }
 
         /**
-         * {@inheritDoc}
-         * @since 2.2
+         * Returns a list of the available addresses.
+         *
+         * @return an array of the available addresses
+         * @deprecated (3.7) overflow if the available addresses are greater than {@code Integer.MAX_VALUE}
          */
-        @Override
-        public String toString() {
-            final StringBuilder buf = new StringBuilder();
-            buf.append("CIDR Signature:\t[").append(getCidrSignature()).append("]")
-                .append(" Netmask: [").append(getNetmask()).append("]\n")
-                .append("Network:\t[").append(getNetworkAddress()).append("]\n")
-                .append("Broadcast:\t[").append(getBroadcastAddress()).append("]\n")
-                 .append("First Address:\t[").append(getLowAddress()).append("]\n")
-                 .append("Last Address:\t[").append(getHighAddress()).append("]\n")
-                 .append("# Addresses:\t[").append(getAddressCount()).append("]\n");
-            return buf.toString();
-        }
+        @Deprecated
+        public String[] getAllAddresses() { return new String[0]; }
+
```

Comment:
There is no need to disable this code. If there are too many addresses, it will throw an error, but if not, the response will be correct.
However it probably makes sense to deprecate the code as it won't always work.